packages/server/modules/cli/commands/workspaces/set-plan.ts

```diff
@@ -28,7 +25,7 @@ const command: CommandModule<
       describe: 'Plan to set the status for',
       type: 'string',
       default: 'business',
-      choices: ['business', 'team', 'pro']
+      choices: ['business', 'team', 'plus']
```

Comment:
it should probably be [plus, business, unlimited, academia] no?

and maybe we can get that value from a constant if thats not too complicated? to avoid this happening again in the future